cmd/stack_addtorepo.go

```diff
@@ -197,6 +197,14 @@ The updated repository index file is created in  ~/.appsody/stacks/dev.local dir
 			// build up stack struct for the new stack
 			newStackStruct := initialiseStackData(stackID, stackYaml)
 
+			versionArchiveTar := stackID + ".v" + stackYaml.Version + ".source.tar.gz"
```

Comment:
@upLukeWeston Suggests that in Windows tar don't exists, he suggests that we need to consider the case for zip files?

Reply:
I don't think this is an issue because we handle tar files on Windows today with the template.tar files.